apps/src/code-studio/pd/application_dashboard/quick_view_table.jsx

```diff
@@ -67,6 +77,19 @@ export default class QuickViewTable extends React.Component {
         ]
       }
     },{
+      property: 'notes',
+      header: {
+        label: 'Notes'
+      },
+      cell: {
+        format: this.formatNotesTooltip,
+        transforms: [
+          (status) => ({
+            style: {...styles.notesCell}
+          })
```

Comment:
`status` here isn't used. Should it be? Otherwise, `() => ...`

Reply:
good catch, no it doesn't need to be there.